log/logger.go

```diff
@@ -123,18 +141,20 @@ func NewLogger(dst io.Writer, options ...Option) Logger {
 		logger = logger.With().Timestamp().Logger()
 	}
 
-	if logCfg.Level != zerolog.NoLevel {
-		logger = logger.Level(logCfg.Level)
-	}
```

Comment:
Not sure why this was here - it wasn't documented anywhere, but I set the defaultConfig value to the zerolog default of `Trace` so that this isn't necessary.